psi4/src/psi4/libfock/CompositeJK.cc

```diff
@@ -532,1059 +291,4 @@ void CompositeJK::compute_JK() {
 
 void CompositeJK::postiterations() {}
 
-// build the J matrix using Weigend's integral-direct density fitting algorithm
```

Comment:
Function moved to `DirectDFJ::build_G_component()` function in libfock/DirectDFJ.cc.